SplitShow/SplitShowController.m

```diff
@@ -88,6 +88,8 @@ - (BOOL)readFromURL:(NSURL *)file error:(NSError *__autoreleasing *)error
 
         self.presentationMode  = [self getPresentationModeForSlideMode:self.presentation.slideMode];
     }
+  
+    self.timerState = 0;
```

Comment:
I think there is no need to set it to zero at this point. You added initialisations to all other relevant places.